### DISCUSSION TITLE
PWA Orientation Settings

### DISCUSSION THREAD

**Discussion by DunklerPhoenix** [photoprism/photoprism#4641] [Q&A]
Heho
Is it possible to configure the PWA orientation?
The automatic orientation is always disabled on my phone (android 15), but the pwa always changes the orientation automatically. Is there somewhere an option that I overlooked to turn this behaviour off?

Greetings
Phoenix

**Comment by omerdduran** [comment]
Hi. PhotoPrism does not have a direct setting that controls the PWA screen rotation behavior. This is probably not a problem with PhotoPrism, but with the browser behavior. Alternatively, you can try a different browser

**Reply by DunklerPhoenix** [reply]
Hmm if I understand the definition here correctly the orientation should be natural and not any.
https://developer.mozilla.org/en-US/docs/Web/Manifest/orientation#examples

The free rotation happens for me with chrome and brave.
